content/en/serverless/installation/ruby.md

```diff
@@ -78,8 +74,123 @@ datadog-ci lambda instrument -f my-function -f another-function -r us-east-1 -e
 
 More information and additional parameters can be found in the [CLI documentation][2].
 
+### Install the Lambda Library
+
+The Datadog Lambda Library can be installed as a layer or a gem. For most functions, Datadog recommends installing the library as a layer. If your Lambda function is deployed as a container image, you must install the library as a gem.
+
+The minor version of the `datadog-lambda` gem always matches the layer version. For example, datadog-lambda v0.5.0 matches the content of layer version 5.
+
+#### Using the layer
+
+[Configure the layers][3] for your Lambda function using the ARN in the following format.
+
+```
+# For regular regions
+arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-<RUNTIME>:{{< latest-lambda-layer-version layer="ruby" >}}
+
+# For us-gov regions
+arn:aws-us-gov:lambda:<AWS_REGION>:002406178527:layer:Datadog-<RUNTIME>:{{< latest-lambda-layer-version layer="ruby" >}}
+```
+
+The available `RUNTIME` is `Ruby2-7`. For example:
```

Comment:
Since there is only one possible runtime, I think we can clean this up and just put the runtime directly in the ARNs.